File: week1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // get height so long as between 1 and 8 inclusive
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    //loop for rows
    int i;
    for (i = 0; i < height; i++)
    {

        //loop for spaces
        int j;
        for (j = i + 1; j < height; j++)
        {
            printf(" ");
        }

        //loop for hashes
        int k;
        for (k = 0; k <= i; k++)
        {
            printf("#");
        }

        //print space in middle
        printf("  ");

        //loop for hashes
        for (k = 0; k <= i; k++)
        {
            printf("#");
        }

        printf("\n");
    }

}

File: week6/mario.py
# Import get_int from CS50
from cs50 import get_int

# Initialise height
height = 0

# Get user input
while (height < 1 or height > 8):
    height = get_int("Height: ")

# Print rows of pyramid
for i in range(1, height+1):

    # Print initial spaces/indents
    print(" " * (height - i), end="")

    # Print first #
    for j in range(i):
        print("#", end="")

    # Print middle space
    print("  ", end="")

    # Print final #
    for k in range(i):
        print("#", end="")

    # Print new line
    print("")
